applications/portal/backend/api/views/rest/experiment.py

```diff
@@ -217,3 +224,73 @@ def download_populations(self, request, pk=None, active_populations=None):
 
     def perform_create(self, serializer):
         experiment = serializer.save(owner=self.request.user)
+
+    def check_if_user_is_owner(self, kwargs_experiment_id):
+        try:
```

Comment:
@D-GopalKrishna I created many months ago a django manager for experiments called `my_experiments` please make use of Django managers to query with pre defined filters

in this case I would do like this (user comes from `request.user`)
```
def check_if_user_is_woner(self, user, experiment_id):
    i_am_the_owner = Experiment.objects.my_experiments(user).filter(id=experiment_id).exists()
```

Reply:
actually I would check this business rule on the save() of the object which then also will be validated in the admin pages

Reply:
Thanks @zsinnema! Can you please elaborate on what you mean by `admin pages`?

Reply:
the django admin